docs/bamboo/index.md

```diff
@@ -0,0 +1,53 @@
+| [Checkout](plugins/Checkout.md)                  | actions/checkout                                                  |
+| [Clean](plugins/Clean.md)                        | run                                                               |
+| [Command](plugins/Command.md)                    | run                                                               |
+| [Docker CLI](plugins/Dockercli.md)               | run                                                               |
+| [Fastlane](plugins/Fastlane.md)                  | ruby/setup-ruby, run                                              |
+| [Grails](plugins/Grails.md)                      | actions/setup-java, run, EnricoMi publish-unit-test-result-action |
+| [Grunt](plugins/Grunt.md)                        | actions/setup-node, run                                           |
+| [Gulp](plugins/Gulp.md)                          | actions/setup-node, run                                           |
+| [Inject Variables](plugins/InjectVariables.md)   | run                                                               |
+| [Junit](plugins/Junit.md)                        | EnricoMi/publish-unit-test-result-action                          |
+| [Maven](plugins/Maven.md)                        | actions/setup-java                                                |
+| [Mocha](plugins/Mocha.md)                        | EnricoMi/publish-unit-test-result-action                          |
+| [Ms Build](plugins/MsBuild.md)                   | run                                                               |
+| [MS Test](plugins/MSTest.md)                     | EnricoMi/publish-unit-test-result-action                          |
+| [Node](plugins/Node.md)                          | run                                                               |
+| [Npm](plugins/Npm.md)                            | actions/setup-node, run                                           |
+| [NUnit Parser](plugins/NunitParser.md)           | EnricoMi/publish-unit-test-result-action                          |
+| [NUnit Runner](plugins/NUnitRunner.md)           | microsoft/vstest-action, EnricoMi/publish-unit-test-result-action |
+| [Repository Branch](plugins/RepositoryBranch.md) | run                                                               |
+| [Repository Commit](plugins/RepositoryCommit.md) | run                                                               |
+| [Repository Push](plugins/RepositoryPush.md)     | run                                                               |
+| [Repository Tag](plugins/RepositoryTag.md)       | run                                                               |
+| [Scp](plugins/Scp.md)                            | appleboy/scp-action                                               |
+| [SSH](plugins/SSH.md)                            | shimataro/ssh-key-action, run                                     |
```

Comment:
```suggestion
| [SSH](plugins/ssh.md)                            | shimataro/ssh-key-action, run                                     |
```

Reply:
Ok it currently is lowercase but should at least first letter be capitalized to match the other providers?

Reply:
I matched it to the others